### PR TITLE
CLIO-41026: Update CSS for thCheckbox to match new designs

### PR DESCRIPTION
**Pull request by brainoreally**

### CODE REVIEW

File: themis_components/thCheckbox/index.scss [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
   vertical-align: sub;
   width: $size;
   height: $size;
-  font-weight: 100;
-  border-radius: $border;
-  background: #fff;
-  border: solid $border $dark-grey-4;
+  background-color: #fff;
+  border-radius: $border-radius;
+	border: solid $border $subtext-color;
```
Comment (brainoreally) [inline themis_components/thCheckbox/index.scss]: I really don't get why git is showing the indentation being weird here and in some other spots, looks fine in my editor?

image

File: themis_components/thCheckbox/examples/2 - Form/html.html [hunk 2]
```diff
@@ -3,10 +3,12 @@
 
     <form action="/echo" method="POST">
       <th-input type="text" with-label="Name" name="name" icon="user"></th-input>
-      <th-checkbox with-label="Admin User" name="isAdmin" ng-model="demo.checked"></th-checkbox>
+      <th-checkbox with-label="Admin User" name="isAdmin" ng-model="demo.checked1"></th-checkbox>
+      <th-checkbox with-label="Enabled" name="enabled" ng-model="demo.checked2"></th-checkbox>
```
Comment (brainoreally) [inline themis_components/thCheckbox/examples/2 - Form/html.html]: just putting some extra checkboxes in this example so it shows spacing between them

File: themis_components/thWithLabel/index.scss [hunk 3]
```diff
@@ -2,19 +2,27 @@ label.th-label {
   @include box-sizing;
   position: relative;
   display: block;
-  margin-bottom: 25px;
+  margin-left: 8px;
```
Comment (seanhealy) [inline themis_components/thWithLabel/index.scss]: Did you check this with labels that aren't inline?

Comment (brainoreally) [inline themis_components/thWithLabel/index.scss]: if so, I think it looks fine ```¯\_(ツ)_/¯```

File: themis_components/thCheckbox/index.scss [hunk 4]
```diff
@@ -34,15 +44,14 @@
   i {
     display: block;
     position: absolute;
-    top: 0px;
+    top: 4px;
     bottom: 0px;
-    left: 0px;
+    left: 2.5px;
```
Comment (seanhealy) [inline themis_components/thCheckbox/index.scss]: Talked to @gthierman about this. We should bring this down to `2px`. 😀

**Pull request merged**